.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 *Issue #, if available:*
 
-*Description of changes:*
+*Description of changes: [Range over map](https://github.com/aws/amazon-ecs-cli/blob/c7335b0259cc1fe526b751d7bc540730e09df2c7/ecs-cli/modules/compose/ecs/utils/convert_task_definition.go#L48) produces different task definitions.
```

Comment:
We want you to copy this license language from this PR template into the description (or a comment) of your PR. Unless you contribute under Apache 2.0, we can't use your work. 

This pull request template is not meant to be changed; since its in the `.github` folder, when you submit a PR against the ECS CLI, the stuff in the PR template will automatically be added to the description of the PR on Github. This didn't happen when you submitted your PR because we only just added the template.

So please leave this file unchanged.

Reply:
Ah, corrected.  Thanks.